### PR TITLE
Add actions performance testing numbers for GHES 3.6

### LINKED ISSUE #19671: Add GHES 3.6 Performance Numbers
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

getting-started-with-github-actions-for-github-enterprise-server.md

### What part(s) of the article would you like to see updated?

The performance numbers matrix for vCPU to connected runners. Also note the new definition of connected runners

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by bquantump**
### Why:

Closes [19671](https://github.com/github/docs/issues/19671)

### What's being changed (if available, include any code snippets, screenshots, or gifs):

image

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by bquantump** [comment]
@brcrista @GitLofquist

File: content/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server.md [hunk 1]
```diff
@@ -80,6 +88,24 @@ Maximum concurrency was measured using multiple repositories, job duration of ap
...
+
+{%- ifversion ghes = 3.6 %}
+
+{% data reusables.actions.hardware-requirements-3.6 %}
+
+{% data variables.product.company_short %} measured maximum connected runners using multiple repositories, job duration of approximately 10 minutes, and 10 MB artifact uploads. You may experience different performance depending on the overall levels of activity on your instance.
+
+{% note %}
+
+**Note:** Beginning with {% data variables.product.prodname_ghe_server %} 3.6, {% data variables.product.company_short %} documents connected runners as opposed to concurrent runners. Connected runners resprents the most runners you can connect and expect to utilize.
```
Comment (brcrista) [inline content/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server.md:100]:
```suggestion
**Note:** Beginning with {% data variables.product.prodname_ghe_server %} 3.6, {% data variables.product.company_short %} documents connected runners as opposed to concurrent jobs. Connected runners represents the most runners you can connect and expect to utilize.
```

We also might want to add a sentence like "Connecting more runners than you expect to utilize can negatively impact performance."

File: content/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server.md [hunk 2]
```diff
@@ -33,14 +33,22 @@ This article explains how site administrators can configure {% data variables.pr
...
 
 The CPU and memory resources available to {% data variables.product.product_location %} determine the number of jobs that can be run concurrently without performance loss. {% data reusables.actions.minimum-hardware %}
 
 The peak quantity of concurrent jobs running without performance loss depends on such factors as job duration, artifact usage, number of repositories running Actions, and how much other work your instance is doing not related to Actions. Internal testing at GitHub demonstrated the following performance targets for GitHub Enterprise Server on a range of CPU and memory configurations:
 
 {% endif %}
 
+{%- ifversion ghes > 3.5 %}
+
+The CPU and memory resources available to {% data variables.product.product_location %} determine the number of connected runners that can be obtained without performance loss. {% data reusables.actions.minimum-hardware %}
```
Comment (brcrista) [inline content/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server.md:47]: "Obtaining" runners doesn't sound right to me.  Maybe:

```suggestion
The CPU and memory resources available to {% data variables.product.product_location %} determine the number of runners that can be configured without performance loss. {% data reusables.actions.minimum-hardware %}
```

**Comment by cmwilson21** [comment]
👋 @bquantump. I just made a tiny edit in the original post so that it links with the issue you opened. Since this is still a draft, I'll hold off on triaging it until you are ready for our team to review. ⚡ 

Thank you! 💛

**Comment by bquantump** [comment]
@cmwilson21
Ready for review 😃

**Comment by cmwilson21** [comment]
Thanks @bquantump! I'll get this triaged for review ⚡

**Comment by up150296365** [comment]
> 
> 
> ### Why:
> 
> Closes [19671](https://github.com/github/docs/issues/19671)
> 
> 
> 
> ### What's being changed (if available, include any code snippets, screenshots, or gifs):
> 
> image
> 
> 
> ### Check off the following:
> 
> - [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
> - [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).
> 
> ### Writer impact (This section is for GitHub staff members only):
> 
> - [ ] This pull request impacts the contribution experience
>   - [ ] I have added the 'writer impact' label
>   - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")
> 
> 
> 
> 77989bf812cbbd6d153d93e51b058ed5baff9805

**Comment by bquantump** [comment]
@cmwilson21 any update?

File: content/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server.md [hunk 3]
```diff
@@ -80,6 +88,24 @@ Maximum concurrency was measured using multiple repositories, job duration of ap
...
+{% data variables.product.company_short %} measured maximum connected runners using multiple repositories, job duration of approximately 10 minutes, and 10 MB artifact uploads. You may experience different performance depending on the overall levels of activity on your instance.
+
+{% note %}
+
+**Note:** Beginning with {% data variables.product.prodname_ghe_server %} 3.6, {% data variables.product.company_short %} documents connected runners as opposed to concurrent jobs. Connected runners represents the most runners you can connect and expect to utilize. It should also be noted that connecting more runners than you can expect to utilize can negatively impact performance.
+{% endnote %}
+
+{% note %}
+
+**Note:** Beginning with {% data variables.product.prodname_ghe_server %} 3.5, {% data variables.product.company_short %}'s internal testing uses 3rd generation CPUs to better reflect a typical customer configuration. This change in CPU represents a small portion of the changes to performance targets in this version of {% data variables.product.prodname_ghe_server %}.
```
Comment (mattpollard) [inline content/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server.md:105]: Let's combine these two notes.

```suggestion
**Notes:**

- Beginning with {% data variables.product.prodname_ghe_server %} 3.6, {% data variables.product.company_short %} documents connected runners as opposed to concurrent jobs. Connected runners represents the most runners you can connect and expect to utilize. It should also be noted that connecting more runners than you can expect to utilize can negatively impact performance.

- Beginning with {% data variables.product.prodname_ghe_server %} 3.5, {% data variables.product.company_short %}'s internal testing uses 3rd generation CPUs to better reflect a typical customer configuration. This change in CPU represents a small portion of the changes to performance targets in this version of {% data variables.product.prodname_ghe_server %}.
```

**Review by mattpollard — DISMISSED** [review]
Thanks for these updates, @bquantump and @brcrista 🙏🏻 I'll incorporate a small change and then go ahead and merge this. I see some opportunities to refactor this section to make it easier to maintain, and I'll create an internal follow-up issue to track that work.

**Review by mattpollard — APPROVED** [review]

**Pull request merged**